tests/kernel/workq/critical/testcase.yaml

```diff
@@ -1,5 +1,6 @@
 common:
   tags: kernel
+  timeout: 60
```

Comment:
is the default timeout 60?

Reply:
@povergoing in order not to block this PR, would you mind approving if this is your only concern? Then @manuargue can submit a follow-up PR to remove this line.

Reply:
sure, No problem for me

Reply:
ok, I will submit a follow-up PR for implementing suggestion on this comment https://github.com/zephyrproject-rtos/zephyr/pull/47579#discussion_r917580682